documentation/sphinx/source/api-c.rst

```diff
@@ -504,6 +504,57 @@ An |database-blurb1| Modifications to a database are performed via transactions.
 
    Returns a value where 0 indicates that the client is idle and 1 (or larger) indicates that the client is saturated. By default, this value is updated every second.
 
+.. function:: FDBFuture* fdb_database_get_client_status(FDBDatabase* db)
+
+   Returns a JSON string containing database client-side status information. At the top level the report describes the status of the 
```

Comment:
Which future-get API should a caller use to retrieve the string? `fdb_future_get_value()`?